cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -144,6 +148,132 @@ def _fit_exponential(self) -> Tuple[np.ndarray, np.ndarray]:
         )
 
 
+class ParallelRandomizedBenchmarkingResult:
```

Comment:
nit: `@dataclass(frozen=True)`. this will allow you to remove the custom `__init__`